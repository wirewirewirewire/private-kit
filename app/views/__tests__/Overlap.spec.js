import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Overlap from '../Overlap';

it('renders correctly', () => {
  const tree = renderer
    .create(<Overlap />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
