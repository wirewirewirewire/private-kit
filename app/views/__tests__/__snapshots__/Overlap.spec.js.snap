// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "backgroundColor": "#FFF",
      "color": "#000",
      "flex": 1,
      "flexDirection": "column",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "borderBottomColor": "rgba(189, 195, 199,0.6)",
        "borderBottomWidth": 1,
        "flexDirection": "row",
        "height": 60,
      }
    }
  >
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "height": 60,
          "opacity": 1,
          "paddingLeft": 20,
          "paddingTop": 21,
          "width": 60,
        }
      }
    >
      <Image
        source={
          Object {
            "testUri": "../../../app/assets/images/backArrow.png",
          }
        }
        style={
          Object {
            "height": 18,
            "width": 18.48,
          }
        }
      />
    </View>
    <Text
      style={
        Object {
          "fontFamily": "OpenSans-Bold",
          "fontSize": 24,
        }
      }
    >
      Check Overlap
    </Text>
  </View>
  <MapView
    initialRegion={
      Object {
        "latitude": 36.56,
        "latitudeDelta": 50,
        "longitude": 20.39,
        "longitudeDelta": 50,
      }
    }
    provider="google"
    style={
      Object {
        "alignSelf": "center",
        "flex": 1,
        "flexDirection": "column",
        "padding": 15,
        "textAlignVertical": "top",
        "width": "96%",
      }
    }
  />
  <View
    style={
      Object {
        "alignSelf": "center",
        "flex": 1,
        "flexDirection": "column",
        "padding": 15,
        "textAlignVertical": "top",
        "width": "96%",
      }
    }
  >
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignSelf": "center",
          "backgroundColor": "#665eff",
          "borderRadius": 12,
          "height": 52,
          "justifyContent": "center",
          "marginTop": 15,
          "opacity": 1,
          "width": 589.9499999999999,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#ffffff",
            "fontFamily": "OpenSans-Bold",
            "fontSize": 14,
            "letterSpacing": 0,
            "lineHeight": 19,
            "textAlign": "center",
          }
        }
      >
        Check Public Data
      </Text>
    </View>
    <Text
      style={
        Object {
          "fontFamily": "OpenSans-Regular",
          "fontSize": 16,
          "lineHeight": 24,
          "marginTop": 12,
        }
      }
    >
      Green circles are clusters of your locations, click to zoom. Red markers are your recorded locations.

Press the button to download public information of confirmed cases, shown as faint purple circles.
    </Text>
  </View>
  <View
    style={
      Object {
        "fontSize": 12,
        "fontWeight": "600",
        "padding": 4,
        "paddingBottom": 10,
        "textAlign": "center",
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "fontFamily": "OpenSans-Regular",
            "fontSize": 16,
            "lineHeight": 24,
            "marginTop": 12,
          },
          Object {
            "paddingTop": 15,
            "textAlign": "center",
          },
        ]
      }
    >
      For more information on the dataset for this map
       
    </Text>
    <Text
      onPress={[Function]}
      style={
        Array [
          Object {
            "fontFamily": "OpenSans-Regular",
            "fontSize": 16,
            "lineHeight": 24,
            "marginTop": 12,
          },
          Object {
            "color": "blue",
            "marginTop": 0,
            "textAlign": "center",
          },
        ]
      }
    >
      github.com/beoutbreakprepared/nCoV2019
    </Text>
  </View>
</RCTSafeAreaView>
`;
